src/ralph/models/edx/converters/xapi/problem_interaction.py

```diff
@@ -0,0 +1,83 @@
+"""Problem interaction event xAPI Converter."""
+
+from ralph.models.converter import ConversionItem
+from ralph.models.edx.problem_interaction.statements import ProblemCheck
+from ralph.models.xapi.assessment.statements import AssessmentAnsweredQuestion
+from ralph.models.xapi.constants import EXTENSION_SESSION_ID
+
+from .base import BaseXapiConverter
+
+
+class ProblemCheckToAssessmentAnsweredQuestion(BaseXapiConverter):
+    """Converts a common edX `problem_check` event to xAPI."""
+
+    __src__ = ProblemCheck
+    __dest__ = AssessmentAnsweredQuestion
+
+    def _get_conversion_items(self):
+        """Returns a set of ConversionItems used for conversion."""
+        conversion_items = super()._get_conversion_items()
+        return conversion_items.union(
+            {
+                ConversionItem(
+                    "object__id",
+                    None,
+                    lambda event: self.platform_url
+                    + "/xblock/block-v1:"
+                    + event["context"]["course_id"]
+                    + event["event"]["problem_id"],
+                ),
+                ConversionItem(
+                    "context__extensions__" + EXTENSION_SESSION_ID,
+                    "session",
+                ),
+                ConversionItem(
+                    "context__contextActivites__definition__type", "context__referer"
```

Comment:
Is the referer in the context field? I can't find it in the edX spec (or in our edX models), it seems to be in the BaseEdxModel alongside (and not inside) the context field.